encrypted-media-respec.html

```diff
@@ -3809,17 +3809,18 @@
         <section class="informative">
           <h4>Potential Attacks</h4>
           <p>Malicious pages could host legitimate applications in an iframe in an attempt hide an attack or deceive the user as to the source, such as making the use appear to be from a legitimate content provider.
-            This is especially relevent for user agents that support have <a href="#security-prompts">user alerts, prompts, and/or permissions</a>.
+            This is especially relevent for implementations that <a href="#security-consent">inform the user or require consent</a>, such as for security and/or <a href="#privacy-consent">privacy reasons</a>.
             In addition to <a href="#network-attacks">Network Attacks</a>, attackers could try to exploit legitimate uses of the APIs defined in this specification by hosting them in an <code>iframe</code>.
             By having the legitimate application performing the actions, the attacker can reuse existing granted permissions (or whitelisting) and/or appear to be a legitimate request or use.
           </p>
         </section>
         <section>
           <h4>Mitigations</h4>
-          <p>User agents are RECOMMENDED to base the UI and persistence of <a href="#security-prompts">user alerts, prompts, and/or permissions</a> on the combination of <a def-id="origin"></a> of the top-level <a def-id="document-concept"></a> and the <a def-id="origin"></a> using the APIs defined in this specification.
+          <p>User agents that <a href="#security-consent">inform the user or require consent</a>, including for security and/or <a href="#privacy-consent">privacy reasons</a>,
+            SHOULD base the UI and persistence of consent on the combination of <a def-id="origin"></a> of the top-level <a def-id="document-concept"></a> and the <a def-id="origin"></a> using the APIs defined in this specification.
             This ensures that users are informed of the main document making the request and that persisting a permission for one (legitimate) combination does not inadvertently allow malicious use to go undetected.
           </p>
-          <p>Authors are RECOMMENDED to prevent other entities from hosting their applications in <code>iframe</code>s.
+          <p>Authors SHOULD prevent other entities from hosting their applications in <code>iframe</code>s.
```

Comment:
There is no change in requirements as this is a synonym. This just reads better. A similar change was made on line 3819.